DirectProgramming/DPC++/ParallelPatterns/dpc_reduce/sample.json

```diff
@@ -0,0 +1,30 @@
+ {
+ 	"guid": "ECF6C8EB-753B-4107-AF64-60662CE41726",
+ 	"name": "DPC Reduce",
+ 	"categories": ["Toolkit/Intel® oneAPI Base Toolkit/oneAPI DPC++ Compiler/oneAPI DPC++ Library/CPU and GPU"],
+ 	"description": "It models transform reduce in different ways showing capability of oneAPI.",
+ 	"toolchain": ["dpcpp"],
+ 	"languages": [{
+ 		"cpp": {}
+ 	}],
+ 	"targetDevice": ["CPU", "GPU"],
+ 	"os": ["linux"],
```

Comment:
Missing "ciTests" field in the sample.json.  "ciTests": Please look at simple-add example for reference